department-of-veterans-affairs/next-build
# Add basic metrics collected from nextbuild into datadog, page count

## Description
Relates to https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16955

## Testing done

In order to test the metrics , I created a data-dog dashboard that ingest all the metrics that we now capture. You can see in the dashboard labeled "next-build content release". I verified that after i run a content-release that metrics get ingested into data dog and that are accurate with the data that I am expecting. 

## Screenshots


## QA steps

To verify this change works go to the data dog dashboard after doing a content release and verify the metrics update.

```[tasklist]
```

## Is this PR blocked by another PR?
- Add the `DO NOT MERGE` label
- Add links to additional PRs here:
